server/resp/src/main/java/org/infinispan/server/resp/commands/cluster/CLUSTER.java

```diff
@@ -0,0 +1,41 @@
+package org.infinispan.server.resp.commands.cluster;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespErrorUtil;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.FamilyCommand;
+
+import io.netty.channel.ChannelHandlerContext;
+
+public class CLUSTER extends FamilyCommand {
+
+   private static final RespCommand[] CLUSTER_COMMANDS;
+
+   static {
+      CLUSTER_COMMANDS = new RespCommand[] {
+            new SHARDS()
+      };
+   }
+
+   public CLUSTER() {
+      super(-2, 0, 0, 0);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx, List<byte[]> arguments) {
+      if (arguments.isEmpty()) {
```

Comment:
I would say this can just go into the base class as any family command requires at least 1 argument.

Reply:
Updated.